### PR TITLE
feature: add unit option to nearest-point

### PR DESCRIPTION
**Pull request by mwenko**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Have read [How To Contribute](https://github.com/Turfjs/turf/blob/master/CONTRIBUTING.md#how-to-contribute).
- [x] Run `npm test` at the sub modules where changes have occurred.
- [x] Run `npm run lint` to ensure code style at the turf module level.

### Feature description

- The `nearestPoint` function does not allow to specify the `units` option. It uses under the hood the `distance` function, that accepts such an option. This change adds that option as optional to the `nearestPoint()` function.
- This allows to specify the unit you wanna have returned in the result `properties.distanceToPoint`
- This saves the conversion step of a caller that needs the value in a different unit than kilometers (which is the default)

### Tests

- A test case was added and succeeds.

### CODE REVIEW

**Comment by JamesLMilner** [comment]
@mwenko this seems like a nice improvement and think it is a good candidate for merging.

Can I request two small changes:

- Can you update `types.ts`  to include the units option. So basically this line goes from:

```
const nearest = nearestPoint(targetPoint, points);
````

to:

```
const nearest = nearestPoint(targetPoint, points, { units: "kilometers" });
````

- Can you please run `npm run docs`  to update the docs

After this I think I can approve

File: packages/turf-nearest-point/index.ts [hunk 1]
```diff
@@ -21,6 +21,7 @@ export interface NearestPoint extends Feature<Point> {
  * @name nearestPoint
  * @param {Coord} targetPoint the reference point
  * @param {FeatureCollection<Point>} points against input point set
+ * @param {Object} [options={}] Optional parameters
```
Comment (rowanwins) [inline packages/turf-nearest-point/index.ts:24]: In here we can also add
````
* @param {Object} [options={}] Optional parameters
 * @param {string} [options. units ='kilometers'] the units of the numeric result
````

**Comment by rowanwins** [comment]
Looks really good @mwenko - just one final little thing and then we'll merge :)

File: packages/turf-nearest-point/index.ts (hunk 1 above)
Comment (mwenko) [reply packages/turf-nearest-point/index.ts:24]: ✅ sure, done

**Pull request merged**